docs/hyperopt.md

```diff
@@ -191,8 +191,10 @@ Currently, the following loss functions are builtin:
 
 * `DefaultHyperOptLoss` (default legacy Freqtrade hyperoptimization loss function)
 * `OnlyProfitHyperOptLoss` (which takes only amount of profit into consideration)
-* `SharpeHyperOptLoss` (optimizes Sharpe Ratio calculated on the trade returns)
-* `SharpeHyperOptLossDaily` (optimizes Sharpe Ratio calculated on daily trade returns)
+* `SharpeHyperOptLoss` (optimizes Sharpe Ratio calculated on trade returns relative to **upside** standard deviation)
+* `SharpeHyperOptLossDaily` (optimizes Sharpe Ratio calculated on **daily** trade returns relative to **upside** standard deviation)
```

Comment:
Are you sure about this? 
i mean - the variable is called `up_stdev` - but there seems to be no filter removing the downside ...

Reply:
It just describes how the formula works, "relative to" here means divided by, upside standard deviation.

Reply:
I understand that - but it's not the "upside standard deviation".

from another site:

> It is defined as the difference between the returns of the investment and the risk-free return, divided by the standard deviation of the investment.

it's the standard-deviation of the whole investment - not just of the upside / profits.

Reply:
Oh ya, I got your point, you're absolutely right, I'll correct it.